### ISSUE TITLE
Does this project support Chinese data sets?

### ISSUE THREAD

**Issue by ganlinlu** [prakashpandey9/Text-Classification-Pytorch#14]
I will complete a Chinese emotion analysis task. Can this project complete the task of Chinese emotion analysis?

**Comment by Abhiram4572** [comment]
Yes, you've to train the embedding weights along with the model (in case you don't have embeddings for Chinese words already)
